common/src/api/internal/shared.rs

```diff
@@ -180,6 +180,35 @@ pub struct BgpConfig {
     pub asn: u32,
     /// The set of prefixes for the BGP router to originate.
     pub originate: Vec<Ipv4Network>,
+
+    /// Shaper to apply to outgoing messages.
+    #[serde(default)]
+    pub shaper: Option<String>,
+
+    /// Checker to apply to incoming messages.
+    #[serde(default)]
+    pub checker: Option<String>,
```

Comment:
What's the status of this? Are we still keeping it in the data structures but not exposing it?

Reply:
Yeah. I put a `#[serde(skip)]` on the public API.